connection/connection.go

```diff
@@ -65,12 +69,32 @@ type Option func(o *options)
 // connection got lost. If that callback returns true, the connection
 // is restablished. Otherwise the connection is left as it is and
 // all future gRPC calls using it will fail with status.Unavailable.
+// Only one of ExitOnConnectionLoss and OnConnectionLoss can be used.
 func OnConnectionLoss(reconnect func() bool) Option {
```

Comment:
is anything going to use this option? do we still need it?

Reply:
The test code uses it. I think we should keep it.